app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -37,10 +37,14 @@ export default function DeleteAlert({ refetch, onClose, selectedNoteIds }) {
 
         <div className="ml-4">
           <h3 className="mb-2 text-lg font-medium text-gray-700">
-            Delete {selectedNoteIds.length} notes?
+            {selectedNoteIds.length > 1
+              ? `Delete ${selectedNoteIds.length} notes?`
+              : "Delete Note"}
           </h3>
           <div className="text-sm leading-5 text-gray-500">
-            Are you sure you want to continue? This cannot be undone.
+            {selectedNoteIds.length > 1
+              ? "Are you sure you want to continue? This cannot be undone."
+              : "Are you sure you want to delete the note? All of your data will be permanently removed from our database forever. This action cannot be undone."}
           </div>
         </div>
       </div>
```

Comment:
Instead of using a custom Modal component, please use neetoUI [Alert Component](https://neeto-ui.netlify.app/#/alert)